swarm64_tpc_toolkit/correctness.py

```diff
@@ -66,6 +75,9 @@ def check_correctness(self, stream_id, query_number):
             self.html += f'<p>{msg}</p>'
             return 'Mismatch'
 
+        benchmark_result = benchmark_result.applymap(self.round_json)
```

Comment:
why are we not applying `DataFrame.round`?

Reply:
- because i did not know it exists
- because i copied it over from the area51 repo where pandas is not used.
so i can change that if you like.

Reply:
actually it has to stay this way because we want:
- to not have more than 12 digits total
- to remove anything after 2 decimals
which requires (what i now fixed):
            return '%.12g' % float('%.2f' % obj)


see also http://www.tpc.org/tpc_documents_current_versions/pdf/tpc-h_v2.18.0.pdf

Reply:
I think using `benchmark_result.round(2)` is good enough. We are not TPC-H compliant anyways and there are way far more rounding rules to be regarded.